### DISCUSSION TITLE
New Gatus release ETA?

### DISCUSSION THREAD

**Discussion by sanderob** [TwiN/gatus#881] [Q&A] [answered]
On the master branch, Teams Workflows is implemented as a way of alerting. This code has although not been released (and is not part of latest). How often is latest normally built, and is there an ETA on the next release from the master branch?

**Comment by TwiN** [comment, marked as answer]
I just released [v5.13.0](https://github.com/TwiN/gatus/releases/tag/v5.13.0). There is no specific cadance to releases; I release a new version of Gatus whenever there is a meaningful enough feature pending for release and it has soaked enough time in latest. Alternatively, when people request it, I can speed up that process a bit :)
